README.md

```diff
@@ -1,15 +1,47 @@
-# bfx-api-node-core
+# Bitfinex Modular WSv2 API Library for Node.JS
 
 [![Build Status](https://travis-ci.org/bitfinexcom/bfx-api-node-core.svg?branch=master)](https://travis-ci.org/bitfinexcom/bfx-api-node-core)
 
-Modular Bitfinex Node.JS API library as an alternative to `bitfinex-api-node`, supporting a plugin system.
+Modular Bitfinex Node.JS API library as an alternative to `bitfinex-api-node`, supporting a plugin system. Connection instances are POJOs as opposed to the
+WSv2 class instances returned by `bitfinex-api-node` and are manipulated in a
+functional style. A connection pool manager is also provided for multiplexing.
+
+### Features
```

Comment:
Seems like you skip h2 (##) and go straight to h3 (###) looks a bit out of line imo, but I might be being a little pedantic

Reply:
It's `###` in the spec, and I think `##` looks a bit too large. Can you confirm `###` is OK? Otherwise I'll update the other PRs.